spec/std/time/measure_spec.cr

```diff
@@ -0,0 +1,30 @@
+require "spec"
+
+describe Time::Measure do
+  it "Time.measure" do
+    elapsed = Time.measure { sleep 0.001 }
+    elapsed.should be >= 1.millisecond
+  end
+
+  it "returns elapsed time" do
+    timer = Time::Measure.new
+    previous = timer.elapsed
+
+    5.times do
+      elapsed = timer.elapsed
+      elapsed.should be >= previous
```

Comment:
Maybe add `previous = elapsed` after this line to ensure it is monotonic? Currently, the 3rd measurement might be smaller than the second.